Exec/inputs_compare_std3

```diff
@@ -33,8 +33,9 @@ exchange_parser(x,y,z) = " (x>0.)*(x<8.4e-8)*(y>0.)*(y<8.4e-8)*(z>0.)*(z<8.4e-8)
 anisotropy_parser(x,y,z) = " (x>0.)*(x<8.4e-8)*(y>0.)*(y<8.4e-8)*(z>0.)*(z<8.4e-8)* 40212.16"
 DMI_parser(x,y,z) = "0."
 
+precession = 1 
 demag_coupling = 1
-FFT_solver = 1
+FFT_solver = 0
```

Comment:
I've found lately solver 1 runs much faster on perlmutter (and solver 0 sometimes randomly fails, not sure why).  Can you change this back in another PR?